.github/workflows/scripts/jira_helper.py

```diff
@@ -296,6 +303,29 @@ def _update_issue(self, issue_id):
             )
             fields['labels'] = gh_issue_labels
 
+        if gh_milestone is not None:
+            jira_fix_version = self._jira.issue_field_value(issue_id, 
+                                                            field="fixVersions")
+            print("JFV", jira_fix_version)
+            jira_version_exists = False 
+            versions = self._jira.get_project_versions_paginated(self._project_key, query=gh_milestone)
+            print(versions["values"])
+            if versions["values"] != []: 
+                jira_fix_version.append(versions["values"][0])
+                fields['fixVersions'] = jira_fix_version
+            else: 
+                new_version = self._jira.add_version(
+                                        self._project_key,
+                                        "",
```

Comment:
I don't really understand.

1. The code [doesn't have a type](https://github.com/atlassian-api/atlassian-python-api/blob/3.41.16/atlassian/jira.py#L2610)
2. It's a [POST to versions](https://github.com/atlassian-api/atlassian-python-api/blob/3.41.16/atlassian/jira.py#L2632-L2634)
3. The REST Api [says `projectId` is an int64](https://developer.atlassian.com/cloud/jira/platform/rest/v2/api-group-project-versions/#api-rest-api-2-version-post) and required.

Reply:
> The REST Api says projectId is an int64 and required.

This is probably my fault as the original author of the script.  I continued to use `project_key` rather than using `project_id`.  It should still work as `project_key` is still in the API but it is deprecated.

Reply:
The response from the `POST` look slike this:

```json
{"archived":false,"description":"An excellent version","id":"10000","name":"New Version 1","project":"PXA","projectId":10000,"releaseDate":"2010-07-06","released":true,"self":"https://your-domain.atlassian.net/rest/api/2/version/10000","userReleaseDate":"6/Jul/2010"}
```